examples/petsc/bpsraw.c

```diff
@@ -809,6 +809,12 @@ int main(int argc, char **argv) {
     ierr = KSPGetIterationNumber(ksp, &its); CHKERRQ(ierr);
     ierr = KSPGetResidualNorm(ksp, &rnorm); CHKERRQ(ierr);
     if (!test_mode || reason < 0 || rnorm > 1e-8) {
+      CeedInt gsize;
```

Comment:
I think we already have gsize floating around and defined. Should we remove these extra two lines in all 4 examples?

Reply:
Done, but it was only in this one example that it was declared in the block scope